<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandChangeVariable</class>
 <widget class="QDialog" name="CommandChangeVariable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="m_pButtonBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Source:</string>
   </property>
  </widget>
  <widget class="QWidget" name="m_pVariableTypeWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QRadioButton" name="m_pInteger">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>integer</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_pString">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>string</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_pBool">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>bool</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_pFloat">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>float</string>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="m_pDestTab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="m_pDestGlobal">
    <attribute name="title">
     <string>Global Variable</string>
    </attribute>
    <widget class="QComboBox" name="m_pGlobalVarBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="m_pDestLocalVariable">
    <attribute name="title">
     <string>Local Variable</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="m_pDestObjectVariable">
    <attribute name="title">
     <string>Object Variable</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QWidget" name="m_pOperatorWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QRadioButton" name="m_pTrigger">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trigger</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_pSet">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_pMultiplicate">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_pModulo">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Modulo</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_pDivide">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_pSubtract">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_pAdd">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="m_pSrcTab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>331</width>
     <height>151</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="m_pSrcValue">
    <attribute name="title">
     <string>Value</string>
    </attribute>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>bool value:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>integer value:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>float value:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>string value:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="m_pValueFloat">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-9999999.999900000169873</double>
     </property>
     <property name="maximum">
      <double>9999999.999900000169873</double>
     </property>
    </widget>
    <widget class="QSpinBox" name="m_pValueInteger">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-2147483647</number>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="m_pValueString">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="m_pValueBool">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAlphabetically</enum>
     </property>
     <item>
      <property name="text">
       <string>false</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>true</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="m_pSrcGlobalVariable">
    <attribute name="title">
     <string>Global Variable</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="m_pSrcLocalVariable">
    <attribute name="title">
     <string>Local Variable</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="m_pSrcObjectVariable">
    <attribute name="title">
     <string>Object Variable</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Destination Variable:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_pButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>CommandChangeVariable</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>CommandChangeVariable</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
